tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclJvmModelInferrer.xtend

```diff
@@ -193,6 +200,18 @@ class TclJvmModelInferrer extends AbstractModelInferrer {
 			annotations += annotationRef('org.junit.Test') // make sure that junit is in the classpath of the workspace containing the dsl
 			body = [element.generateMethodBody(trace(element, true))]
 		]
+
```

Comment:
I would prefer to extract methods to create the test method and the macro methods.